components/FeedbackButton/FeedbackButton.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import { AnimatePresence, motion } from 'framer-motion';
+import ButtonTooltip from './ButtonTooltip';
+import { useEffect, useState } from 'react';
+import ClickUpFormModal from './FeedbackModal';
+
+const FeedbackButton = () => {
+  const [showTooltip, setShowTooltip] = useState(true);
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setShowTooltip(false);
+    }, 10000); // tooltip disappears after 10 seconds
+
+    return () => clearTimeout(timeoutId);
+  }, []);
+
+  return (
+    <>
+      <ClickUpFormModal open={open} setOpen={setOpen} />
+      <motion.div
+        initial={{ opacity: 1 }}
+        animate={{ x: '25px', rotate: -60 }}
```

Comment:
In my opinion the rotated button here looks a bit odd. Is the idea just to move it out of the way after the tooltip disappears? I think it would be fine to animate it to the bottom corner without rotating.

Reply:
It looks creepy, right? 😅 I'll fix it.

Reply:
I put the feedback button in a top-level banner according to Joshua's suggestion.

Reply:
The top-level banner looks awesome!